src/Common/Commands.Common.Authentication/Properties/Resources.resx

```diff
@@ -142,7 +142,7 @@
     <value>The environment name '{0}' is not found.</value>
   </data>
```

Comment:
Changes to common code should include an addition to the ChangelOg for the Profile module

Reply:
done